en/animation/animation-editor.md

```diff
@@ -0,0 +1,178 @@
+# Get Familiar with the Animation Editor
+
+The **Animation** panel panel is used to edit and preview the animation clip of the currently selected node. When editing animation data or related properties, it will take effect immediately after mouse focus leaves.
```

Comment:
```suggestion
The **Animation** panel is used to edit and preview the animation clip of the currently selected node. When editing animation data or related properties, it will take effect immediately after mouse focus leaves.
```